src/main/drivers/mco.c

```diff
@@ -38,8 +38,15 @@ void mcoInit(const mcoConfig_t *mcoConfig)
     if (mcoConfig->enabled[1]) {
         IO_t io = IOGetByTag(DEFIO_TAG_E(PC9));
         IOInit(io, OWNER_MCO, 2);
+#if defined(STM32F7) || defined(STM32H7)
```

Comment:
Can we keep STM32H7 out of here until we actually add support for it to Betaflight? ;-)

Reply:
Ah. My bad. I was doing this on H7 branch 😅